src/App.tsx

```diff
@@ -14,11 +15,88 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+interface Option {
+  [key: string]: boolean;
+}
+
+type State = {
+  goods?: string;
+  visited: Option;
+};
+
+class App extends React.Component<{}, State> {
+  state: State = {
+    visited: { Jam: true },
```

Comment:
```suggestion
    selected: ['Jam'],
```
Сделай это свойство массивом, и не надо будет использовать Object.keys